spec/declaration.dd

```diff
@@ -473,8 +473,9 @@ alias abc = foo.bar; // is it a type or a symbol?
 struct S { static int i; }
 S s;
 
-alias a = s.i; // illegal, s.i is an expression
-alias b = S.i; // ok
+alias a = s.i; // ok, s.i is a symbol
+alias b = S.i; // ok, S.i also is a symbol
```

Comment:
`alias b = s.i` seems redundant and doesn't illustrate anything new.  I suggest maybe adding a member `static int j;` to `S` and use that instead to illustrate the idea, or something like that.

Reply:
>     a = 2; // sets s.i to 2

Doesn't work like that. An alias of `s.i` is the same as one of `S.i`. It doesn't carry `this` with it. It can't be used to access `s.i`.

Reply:
Oh yeah, of course 🤦‍♂️ I suppose I was bit too quick here.

Reply:
Or should we simplify the example even more? After all, it is about alias not working on expressions. Structs are not necessary to show this:
```D
int a;

alias b = a; // ok, a is a symbol
alias c = a + a; // illegal, a + a is an expression
```